tst/EnergyPlus/unit/SurfaceGeometry.unit.cc

```diff
@@ -1055,9 +1055,9 @@ TEST_F(EnergyPlusFixture, SurfaceGeometryUnitTests_isAlmostEqual3dPt)
 
     EXPECT_TRUE(isAlmostEqual3dPt(a, b));
 
-    b.x = 7.01;
-    b.y = 11.01;
-    b.z = 17.01;
+    b.x = 7.0095;
+    b.y = 11.0095;
+    b.z = 17.0095;
```

Comment:
Adjusted this unit test due to the tighter tolerance in these functions (1cm instead of 1.27cm).